exporter/datadogexporter/traces_exporter.go

```diff
@@ -117,6 +117,9 @@ func (exp *traceExporter) consumeTraces(
 	tags := make(map[string]struct{})
 	for i := 0; i < rspans.Len(); i++ {
 		rspan := rspans.At(i)
+		if noAPMStatsFeatureGate.IsEnabled() {
+			rspan.Resource().Attributes().PutStr("_dd.stats_computed", "true")
```

Comment:
With doing this, I think you need to mark the datadog trace exporter as `MutatesData: true` similar to metrics: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/7a08a6f86cc32f86c86fb58e661f8584d53613f3/exporter/datadogexporter/factory.go#L279

Reply:
Good point. Done.

Reply:
P.S. It can be done without mutating data, but requires a small change on the trace-agent side too. LMK if you think it would be better to do it that way.

Reply:
Yeah I think it'd be better to do it in trace-agent and not mutate the input pdata. `MutatesData` means the exporter may need to copy the pdata and may impact performance / throughput.